.github/workflows/network.yml

```diff
@@ -0,0 +1,70 @@
+name: Network CI
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'lib/vdsm/network/**'
+      - 'lib/vdsm/common/network/**'
+      - 'tests/network/**'
+      - '.github/workflows/network.yml'
+  pull_request:
+    paths:
+      - 'lib/vdsm/network/**'
+      - 'lib/vdsm/common/network/**'
+      - 'tests/network/**'
+      - '.github/workflows/network.yml'
+
+jobs:
+  unit:
+    env:
+      IMAGE_TAG: ${{ matrix.tag }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tag: [ centos-8, centos-9 ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt install podman
+      - name: Run unit tests
+        run: sudo -E ./tests/network/unit/run-tests.sh
+
+  integration:
+    env:
+      IMAGE_TAG: ${{ matrix.tag }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tag: [ centos-8, centos-9 ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt install podman
+      - name: Run integration tests
+        run: sudo -E ./tests/network/integration/run-tests.sh
```

Comment:
All the jobs have the same configuration, it looks like the only change is the name of the script. Can we use a matrix that use the same configuration to run "unit", "integration", and "functional" tests?

Reply:
Unit and integration can be matrix, functional need additional argument.